.buildkite/commands/lint-localized-strings-format.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -eu
+
+echo "--- :rubygems: Setting up Gems"
+install_gems
+
+echo "--- :cocoapods: Setting up Pods"
+install_cocoapods
```

Comment:
This line made me go down a rabbit hole thinking about whether we can optimize how we generate the pods cache further #19551.